oszillator-sfm.py

```diff
@@ -13,7 +13,7 @@
 fps = 16      # frames per second
 dt = 0.001    # [s] integrator step length
 t_end = 10    # [s] integration time
-N_ped = 2     # number of pedestrians
+N_ped = 3     # number of pedestrians
```

Comment:
why do you change the number of pedestrians to 3? 2 is just fine.

Reply:
You can have two agents there, but the first one in the figure is just like a wall (The red line).  So It's better to define 3 persons there.  Based on your model the agent is only affected by the one in front.  So the second agent is oscillating with the first one who stand still (like a wall).  The third one is oscillating with the second one who also oscillates.  So there is some difference between 2 agents and 3 agents.  There is no more difference if you have more than 3 agents, I think.